Firestore/core/src/firebase/firestore/util/status.h

```diff
@@ -63,6 +65,8 @@ class ABSL_MUST_USE_RESULT Status {
 
 #if defined(__OBJC__)
   static Status FromNSError(NSError* error);
+
+  NSError* ToNSError() const;
```

Comment:
Hm. I wonder if these should be free functions, living alongside MakeNSError() etc in error_apple.h?

No action required.